libopeniscsiusr/idbm.h

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (C) 2017-2018 Red Hat, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Author: Gris Ge <fge@redhat.com>
+ */
+
+#ifndef __ISCSI_OPEN_USR_IDBM_H__
+#define __ISCSI_OPEN_USR_IDBM_H__
+
+#include <stdio.h>
+#include <stdbool.h>
+
+#include "libopeniscsiusr/libopeniscsiusr_common.h"
+
+#define ISCSI_CONFIG_ROOT	"/etc/iscsi/"
+#define IFACE_CONFIG_DIR	ISCSI_CONFIG_ROOT"ifaces"
+
+struct __DLL_LOCAL idbm;
+
+struct idbm {
+	int		refs;
+} idbm_t;
```

Comment:
It looks like this was created with the intention of doing a `typedef`, but incomplete?  You are creating an instance of `struct ibdm` here which isn't being used.

Reply:
I forgot to remove the `idbm_t`.
Thanks.